pyproject.toml

```diff
@@ -19,7 +19,7 @@ onnx = "1.17.0"
 onnxruntime = "1.18.1"
 pandas = "2.2.3"
 python = ">=3.10,<4.0"
-pyroengine = { git = "https://github.com/pyronear/pyro-engine.git", rev = "main"}
+pyroengine = { git = "https://github.com/pyronear/pyro-engine.git", rev = "develop"}
```

Comment:
I am curious why we now use `develop` instead of `main`? Do we need to switch back to `main` once some changes landed in the `main` branch?